### PR TITLE
Blog post - GC elections 2022

### PR DESCRIPTION
**Pull request by jpkrohling**
Blog post announcing the GC Elections 2022

Closes https://github.com/open-telemetry/community/issues/1166

Signed-off-by: Juraci Paixão Kröhling <juraci@kroehling.de>

### CODE REVIEW

**Comment by jpkrohling** [comment]
@open-telemetry/governance-committee, could you please do a final review before this gets merged?

**Review by bhs — APPROVED** [review]
thank you!

**Review by lizthegrey — APPROVED** [review]

**Comment by lizthegrey** [comment]
approved, but also, I respectfully decline the invitation to run for another term :)

(which then makes it not a conflict of interest for me to help with elections this year)

**Review by dyladan — APPROVED** [review]

File: content/en/blog/2022/gc-elections-2022.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+---
+
+The OpenTelemetry project is excited to announce the 2022 OpenTelemetry
+Governance Committee (GC) election. Nominations are due by end-of-day on 7
+October 2022, with the ratification happening by 8 October 2022 and a list of
+eligible candidates will be shared on 9 October 2022. Voting will take place
+between 18 October 2022 and 20 October 2022, and the final election results will
+be announced 22 October 2022.
+
+## Vote!
```
Comment (cartermp) [inline content/en/blog/2022/gc-elections-2022.md:15]: Heh, on mobile when I read this, I thought it said `Votel` and I was going to applaud the pun.

**Review by cartermp — APPROVED** [review]

**Comment by cartermp** [comment]
@open-telemetry/blog-approvers please take a look!

File: content/en/blog/2022/gc-elections-2022.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
+---
+title: Announcing the 2022 OpenTelemetry Governance Committee Election
+linkTitle: OpenTelemetry Governance Committee Election 2022
```
Comment (svrnm) [inline content/en/blog/2022/gc-elections-2022.md:3]: Suggestion: Drop OpenTelemetry here to make that title shorter?

File: content/en/blog/2022/gc-elections-2022.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+To learn more about the election process, read about all of the details
+[here](https://github.com/open-telemetry/community/blob/main/elections/2022/governance-committee-election.md).
+
+## Interested in joining the Governance Committee?
+
+If you’ve been working on OpenTelemetry and seeing it grow, or are an end-user
+that wants to help us make OpenTelemetry better, now’s the time to consider
+running for a seat on the Governance Committee. You can read about the
+Governance Committee's role in
+[this blog](https://medium.com/opentelemetry/opentelemetry-governance-committee-explained-860353baba0)
```
Comment (svrnm) [inline content/en/blog/2022/gc-elections-2022.md:44]: Assuming this is going to be referenced in future election announcements as well: Should we move that to blog.opentelemetry.io? (does not have to be part of this PR)

File: content/en/blog/2022/gc-elections-2022.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+invite them to run for re-election if they so choose: Alolita Sharma, Daniel
+Dyla, Liz Fong-Jones and Morgan McLean. After this election, the Governance
+Committee will comprise 9 members with staggered 2-year terms, all elected by
+the community.
+
+## Questions?
+
+For any election related questions, please file an issue on the
+[community repo](https://github.com/open-telemetry/community/issues) here and
+tag @jpkrohling, @bogdandrutu, and @bhs. Or send us a message on CNCF Slack in
```
Comment (svrnm) [inline content/en/blog/2022/gc-elections-2022.md:63]: Suggestion: Put links around the tags, e.g. `[@jpkrohling](https://github.com/jpkrohling)`

Comment (svrnm) [inline content/en/blog/2022/gc-elections-2022.md:63]: CNCF Slack -> `[CNCF Slack](https://slack.cncf.io/)`

File: content/en/blog/2022/gc-elections-2022.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+#opentelemetry if you have an urgent access issue during voting.
```
Comment (svrnm) [inline content/en/blog/2022/gc-elections-2022.md:64]: #opentelemetry -> `[#opentelemetry](https://cloud-native.slack.com/archives/CJFCJHG4Q)`

**Review by svrnm — APPROVED** [review]
Some suggestions, nothing that would block if we want to have this published asap.

**Review by austinlparker — APPROVED** [review]
content lgtm

File: content/en/blog/2022/gc-elections-2022.md [hunk 6]
```diff
@@ -0,0 +1,66 @@
+---
+title: Announcing the 2022 OpenTelemetry Governance Committee Election
+linkTitle: OpenTelemetry Governance Committee Election 2022
+date: 2022-09-14
```
Comment (svrnm) [inline content/en/blog/2022/gc-elections-2022.md:4]: @jpkrohling , let's update the date to 2022-09-15 and then we can merge & publish!

File: content/en/blog/2022/gc-elections-2022.md (hunk 3 above)
Comment (jpkrohling) [reply content/en/blog/2022/gc-elections-2022.md:44]: Created https://github.com/open-telemetry/opentelemetry.io/issues/1752 to track this.

**Comment by jpkrohling** [comment]
Thank you for the suggestions. I applied them all and updated this PR.

**Comment by svrnm** [comment]
@jpkrohling: I merged it, should be live in a few minutes.

**Comment by svrnm** [comment]
Here we go: https://opentelemetry.io/blog/2022/gc-elections-2022/

**Pull request merged**